src/platform_impl/android/mod.rs

```diff
@@ -996,8 +1000,8 @@ pub struct MonitorHandle {
     app: AndroidApp,
 }
 impl PartialOrd for MonitorHandle {
-    fn partial_cmp(&self, _other: &Self) -> Option<std::cmp::Ordering> {
-        Some(std::cmp::Ordering::Equal)
+    fn partial_cmp(&self, other: &Self) -> Option<std::cmp::Ordering> {
+        Some(self.cmp(other))
     }
```

Comment:
This seems like an unintended change?

Reply:
That was a clippy error that I just fixed transparently.

Reply:
But it may be wrong now? Or at least the `Ord` implementation should also be fixed then.

Reply:
Have you seen the change? It's an equivalent transformation, it was always `Eq` before, now I call `cmp` which is also always `Eq`. The `cmp` is a method on `Ord`.

Reply:
Oh sorry, for some reason I had gotten it into my head that it was calling `PartialEq` on the `AndroidApp` field...